app/presenters/contacts_finder_presenter.rb

```diff
@@ -91,7 +91,7 @@ def need_ids
   end
 
   def organisations
-    []
+    [organisation.content_id]
```

Comment:
Probably worth making this defensive so that it won't insert nil values between deployment and the organisation importer being run.